src/bin/index.ts

```diff
@@ -46,7 +46,8 @@ async function main() {
         routes
       }
     },
-    request: {} as unknown as Request
+    request: {} as unknown as Request,
+    appContext: {},
```

Comment:
app context is not available in static build scenarios. maybe a function could be provided to generate it; for cloudflare, at least, there is a way to do this locally. it is left for future work for now.